test/src/test/java/benchmarks/BenchmarkTest.java

```diff
@@ -25,6 +28,8 @@
      */
     @Test
     public void runSampleBenchmark() throws Exception {
+        assumeFalse(Functions.isWindows());
```

Comment:
benchmarks don't work on windows?

Reply:
Dunno, had some problems here and it did not seem important.